core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -190,7 +190,11 @@ <h3>{{ 'I18N_SAVE_PROGRESS_TEXT' | translate }}</h3>
     <p class="copy-text">{{ 'I18N_SAVE_EXPLORATION_PROGRESS_TEXT_5' | translate }}</p>
     <div class="uid-copy" [ngStyle]="{ 'flex-direction': isLanguageRTL() ? 'row-reverse' : 'row' }">
       <div class="unique-progress-id">{{ loggedOutProgressUniqueUrl }}</div>
-      <button type="button" class="btn btn-secondary uid-copy-btn" (click)="copyProgressUrl()">Copy</button>
+      <div class="uid-btn-container">
+        <div class="uid-btn-message-box">Copied!</div>
```

Comment:
Also use the I18N key here, look for similar I18N key examples above! YOu will need to add it in en.json file and its explanantion in qqq.json file! (Since it is a user facing page)

Reply:
done